packages/components/src/textarea-control/index.tsx

```diff
@@ -1,12 +1,13 @@
 /**
  * External dependencies
  */
-import type { ChangeEvent } from 'react';
+import type { ChangeEvent, ForwardedRef } from 'react';
```

Comment:
Nit: could use the `React` global instead of importing the type?

Something like `React.ForwardedRef` and `React.ChangeEvent`. This is mostly useful to avoid getting confused between react synthetic event types and standard DOM event types

Reply:
Thanks for the review!

- In d3eba32, I used React global.
- In e649d10, I forgot to move the code example comment, so I fixed it.